paasta_tools/check_chronos_jobs.py

```diff
@@ -152,31 +144,65 @@ def message_for_status(status, service, instance, cluster):
         raise ValueError('unknown sensu status: %s' % status)
 
 
-def sensu_message_status_for_jobs(chronos_job_config, service, instance, cluster, job_state_pairs):
-    if len(job_state_pairs) > 1:
-        sensu_status = pysensu_yelp.Status.UNKNOWN
-        output = (
-            "Unknown: somehow there was more than one enabled job for %s%s%s.\n"
-            "Talk to the PaaSTA team as this indicates a bug." % (service, utils.SPACER, instance)
-        )
-    elif len(job_state_pairs) == 0:
+class TZ(tzinfo):
+
+    def utcoffset(self, dt):
+        return timedelta(minutes=0)
+
+    def dst(self, dt):
+        return timedelta(minutes=0)
+
+
+utc = TZ()
+
+
+def job_is_stuck(last_run_iso_time, interval_in_seconds):
+    if last_run_iso_time is None or interval_in_seconds is None:
+        return False
+    last_run_datatime = isodate.parse_datetime(last_run_iso_time)
+    return last_run_datatime + timedelta(seconds=interval_in_seconds) < datetime.now(utc)
```

Comment:
If I could, that wouldn't be python2.7.

utcnow() returns UTC date and time with with tzinfo=None and can't be compared with datetime objects that have a valid tzinfo.

```
Python 2.7.6 (default, Jun 28 2016, 17:53:28)
[GCC 4.6.3] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import isodate
>>> from_iso = isodate.parse_datetime('2016-07-26T22:03:00+00:00')
>>> from_iso
datetime.datetime(2016, 7, 26, 22, 3, tzinfo=<FixedOffset '+00:00'>)
>>>
>>> from datetime import datetime
>>> time_now = datetime.utcnow()
>>> time_now
datetime.datetime(2017, 2, 21, 17, 42, 58, 580216)
>>>
>>> time_now > from_iso
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can't compare offset-naive and offset-aware datetimes
>>>
```